go/cmd/dolt/commands/log.go

```diff
@@ -103,8 +103,8 @@ func logWithLoggerFunc(ctx context.Context, commandStr string, args []string, dE
 	return logCommits(ctx, dEnv, cs, loggerFunc, numLines)
 }
 
-func getCommitSpec(dEnv *env.DoltEnv, apr *argparser.ArgParseResults) (*doltdb.CommitSpec, error) {
-	if apr.NArg() == 0 {
+func parseCommitSpec(dEnv *env.DoltEnv, apr *argparser.ArgParseResults) (*doltdb.CommitSpec, error) {
```

Comment:
Perhaps this should be factored out somewhere too now that it's also being used in blame.